python-avd/pyavd/_eos_designs/schema/schema_fragments/platform_settings.schema.yml

```
@@ -127,6 +127,10 @@ keys:
                   convert_types:
                     - str
                   description: Maximum rx_queue count supported on any interface.
+            all_active_multihoming:
```

With the latest changes elsewhere I think we should rename this key. 
```suggestion
            evpn_gateway_all_active_multihoming:
```